### PR TITLE
Doc 6673 improve user permissions and capabilities details

### PR DESCRIPTION
**Pull request by zuluecho9**
See jira for details. Various improvements related to making things clearer for user type differences, capabilities, roles, etc.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx [hunk 1]
```diff
@@ -8,14 +8,11 @@ metaDescription: >-
...
-  - >-
-    /docs/accounts/accounts-billing/new-relic-one-pricing-users/role-based-access-control-rbac
-  - >-
-    /docs/accounts/accounts-billing/new-relic-one-pricing-users/add-manage-users-groups-roles
+  - /docs/accounts/accounts-billing/new-relic-one-pricing-users/organization-level-user-management-roles-groups-access-grants
+  - /docs/accounts/accounts-billing/new-relic-one-pricing-users/manage-user-groups-roles-access-grants
+  - /docs/accounts/accounts-billing/new-relic-one-pricing-users/role-based-access-control-rbac
+  - /docs/accounts/accounts-billing/new-relic-one-pricing-users/add-manage-users-groups-roles
+
 watermark: |-
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx:16]: The watermark is a little odd.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx [hunk 2]
```diff
@@ -623,48 +607,107 @@ For accounts on New Relic One pricing, learn more about [user-related billing ca
...
+      <td>
+        **Group**
+      </td>
+
+      <td>
+        Full users can be assigned to [default groups](#groups) (**User** and **Admin**) or custom groups. 
+      </td>
+
+      <td>
+        Basic users can’t be assigned to default groups. They can be assigned to custom groups, but note that a basic user always has basic user permissions, no more no less, even when in a custom group.
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx:665]: I like the "no more no less" but is it possible to have less permissions than a basic user?

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx [hunk 3]
```diff
@@ -708,95 +755,97 @@ Standard roles include:
...
       </td>
 
       <td>
-        Account
+        Organization
       </td>
 
       <td>
-        Provides ability to manage subscriptions and billing, and read-only access to the rest of the platform.
+        Provides the ability to manage [organization](/docs/accounts/accounts-billing/new-relic-one-pricing-users/new-relic-account-structure/-level settings, including organization structure, name, and preferences. Due to our recent switch to the [New Relic One account/user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models), this role currently has few abilities but more will be added over time.
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx:787]: You're missing a closing parenthesis on that organization link.

**Review by paperclypse — CHANGES_REQUESTED** [review]
There's a broken link in New Relic One user model: Understand the user structure doc. Definitely need to fix that before we can merge this in. I've flagged the specific line with a comment.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx:665]: it is possible but only (nonintuitively) if you're a full user who's been specifically assigned a group with less permissions than a basic user. Does that make sense? So full users can theoretically have less permissions than a basic user. A basic user's permissions never change.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx (hunk 3 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx:787]: god bless you

**Pull request merged**